### ISSUE TITLE
Feature request: Support inserting local images

### ISSUE THREAD

**Issue by twstokes** [blocknotes-org/blocknotes#23]
Images can only be inserted via URL. It would be nice to support local images from the device's library and camera.

<img src="https://github.com/user-attachments/assets/a1170129-49a2-4819-95bb-e850b776bd31" width="300" />

Blocknotes v1.2 / iPhone 12 / iOS 17.4.1

**Comment by ellatrix** [comment]
Yes, I plan to work on local images. I've disabled the image block for now in 4ae89b72e8785c0c6c70e0442f615a1099b27ab3.

**Comment by jeryj** [comment]
Agreed! Local images would be helpful for me too.
